### ISSUE TITLE
Landing page for current issue has no connection to full-issue content

### ISSUE THREAD

**Issue by baechtel** [McFateM/rootstalk-static#9]
The current issue's landing page doesn't really indicate that there is content available in the issue, other than the three content-pieces which are listed as "featured articles."

**Comment by McFateM** [comment]
I think it may be more obvious if/when there's more than one issue banner/link up front, if that is what's intended?  In any case, it might be wise to add some text over the image, maybe something like this:

![Rootstalk Example](https://user-images.githubusercontent.com/8472769/85328462-602edf00-b496-11ea-8240-4175c6c26b12.png)

**Comment by baechtel** [comment]
Agreed, Mark. Certainly that can be a "for-now" solution, unless Jeremy has something to suggest which is more in keeping with his design. I'm cc'ing him on this reply; hopefully he'll weigh in on this discussion.

Cheers,

Mark B.

Mark Baechtel
Editor, Rootstalk: A Prairie Journal of Culture, Science and the Arts
Lecturer and Director of Forensic Activities
Grinnell College
Grinnell, IA 50112

________________________________
